scripts/create-test-users.js

```diff
@@ -4,7 +4,7 @@ const factory = require('../test/api/support/factory');
 const { authenticatedSession } = require('../e2e/auth-session');
 
 async function createUsers() {
-  const user = await factory.user({ username: 'test-e2e-user' });
+  const user = await factory.user({ username: process.env.PAGES_TEST_USER || 'generic-test-user' });
```

Comment:
Do we want to update this user to belong to an org? Manage an org?

Reply:
we don't really have standardized fixture data for them to test against but having them in an org could be okay